app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -54,14 +62,18 @@ export const PressGallery: React.FC = () => {
   const fetchResultHits = React.useCallback(
     async (filter: MsearchMultisearchBody, name?: string, date?: string) => {
       try {
-        const res: any = await findContentWithElasticsearch(filter, false);
+        const res = await findContentWithElasticsearch(filter, false);
         if (!!name)
           setPressMembers((pressMembers) =>
-            pressMembers.map((c) => (c.name === name ? { ...c, hits: res.hits.total.value } : c)),
+            pressMembers.map((c) =>
+              c.name === name ? { ...c, hits: (res.hits.total as any).value } : c,
+            ),
           );
         if (!!date)
           setDateOptions((dates) =>
-            dates.map((d) => (d.value === date ? { ...d, hits: res.hits.total.value } : d)),
+            dates.map((d) =>
+              d.value === date ? { ...d, hits: (res.hits.total as any).value } : d,
```

Comment:
for some reason it says `hits.total` is of type number when there is a value attribute we need to read

Reply:
It can be a number but also an object.  I'm not sure how to control that.  Probably need a util function to be safe.